style.css

```
@@ -384,4 +384,77 @@
   .im {
     margin-bottom: -20px;
   }
+ /*contact form section*/
+ .formsection{
+  background-image: url("./img/header-shapes/ mobile.png");
+  background: #6070FF;
```

```suggestion
.formsection{
  display: flex;
  justify-content: center;
  flex-wrap: wrap;
  border-top-left-radius: 90px;
  background-image: url("img/contact\ form\ shapes-mobile@2x.png");
  background-size: 100%;
  background-color: #6070FF;
  width: 100%;
}
```

Try using flexbox in this section for semantic reasons and to fulfill the activities' requirements. It should also help with the styling